code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -101,12 +101,21 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 	if(!.)
 		. += HEAT_CAPACITY_VACUUM //we want vacuums in turfs to have the same heat capacity as space
 
-	/// Calculate moles
+/// Calculate moles
 /datum/gas_mixture/proc/total_moles()
 	var/cached_gases = gases
 	TOTAL_MOLES(cached_gases, .)
 
-	/// Calculate pressure in kilopascals
+/// Checks to see if gas amount exists in mixture.
+/// Do NOT use this in code where performance matters!
+/// It's better to batch calls to garbage_collect(), especially in places where you're checking many gastypes
```

Comment:
Sorry for not mentioning this earlier, but this should be in the [dm doc format](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md), so it's parsed properly <

Reply:
This works, actually. Up to user preference.

Reply:
Last time I saw it tried shit got messed up, but I could be misremembering

Reply:
Brain no work so good, code document well